packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -123,10 +172,12 @@ class Statement extends React.Component {
         return (
             <React.Fragment>
                 <ReportsMeta
-                    i18n_heading={localize('Statement')}
-                    i18n_message={localize(
-                        'View all transactions on your account, including trades, deposits, and withdrawals.'
-                    )}
+                    optional_component={
+                        this.props.landing_company_shortcode === 'iom' ||
+                        this.props.landing_company_shortcode === 'virtual'
```

Comment:
What was the goal that was going to achieve with adding the `virtual` check here?

Reply:
It should be `malta` .. I wrote it to test something and forgot to change.